Jenkinsfile

```diff
@@ -21,19 +21,18 @@
 
 properties(
     [
-        buildDiscarder(logRotator(artifactDaysToKeepStr: env.BRANCH_NAME == 'master' ? '14' : '7',
-                                  artifactNumToKeepStr: '50',
+        buildDiscarder(logRotator(artifactNumToKeepStr: env.BRANCH_NAME == 'master' ? '15' : '5',
                                   daysToKeepStr: env.BRANCH_NAME == 'master' ? '30' : '14',
                                   numToKeepStr: env.BRANCH_NAME == 'master' ? '20' : '10')
-        ),
-        disableConcurrentBuilds()
+        )//,
+        //disableConcurrentBuilds()
     ]
 )
 // final def oses = ['linux':'ubuntu && maven', 'windows':'windows-he']
-final def oses = ['linux':'ubuntu && maven']
-final def mavens = env.BRANCH_NAME == 'master' ? ['3.x.x', '3.6.3'] : ['3.x.x']
+final def oses = ['linux':'ubuntu']
+final def mavens = env.BRANCH_NAME == 'master' ? ['3.x.x', '3.8.8'] : ['3.x.x']
```

Comment:
on GH action we have 3.6.3 still

Reply:
but we support only 3.8.x and 3.9.x as of today :)

Reply:
as minimum maven we have in code 3.6.3 

https://github.com/apache/maven-surefire/blob/66e85890cc4c4fb466726abf7339cbd952052ecc/pom.xml#L91

We can change it to 3.8.8 but if possible I would like to see it in separate PR

By the way should we start discussion about next minimum version for all plugins?

Reply:
I have restored this. If this hurt you :) cced9cd1e557ff6b96da41862421373a4fae5253
But to be honest, I don't understand why we should support a plugin with a core version we do not officially support. This looks weird to me.

Reply:
It is not a problem for me, even more I am for it ... But should be done in separate PR. 

We should discuss and start switching  all plugins

We also have info about 3.6.3 at https://maven.apache.org/docs/history.html